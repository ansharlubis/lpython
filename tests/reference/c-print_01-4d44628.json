{
    "basename": "c-print_01-4d44628",
    "cmd": "lpython --no-color --show-c {infile}",
    "infile": "tests/../integration_tests/print_01.py",
    "infile_hash": "4755131262f8eb382c206da769880635ad00787824c37f788d1c3d44",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "c-print_01-4d44628.stderr",
    "stderr_hash": "5195195ab36535a69b9a9c3cfce3a1fd50e07199b4e80fddcddf1278",
    "returncode": 2
}