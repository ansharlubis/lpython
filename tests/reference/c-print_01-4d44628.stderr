semantic error: The symbol sep is not found
 --> tests/../integration_tests/print_01.py:7:5
  |
7 |     print("a", "b", sep=x)
  |     ^^^^^^^^^^^^^^^^^^^^^^ 
